client/multi-currency/enabled-currencies-list/list-item.js

```diff
@@ -43,7 +43,8 @@ const EnabledCurrenciesListItem = ( {
 				<div className="enabled-currency__flag">{ flag }</div>
 				<div className="enabled-currency__label">{ name }</div>
 				<div className="enabled-currency__code">
-					({ symbol } { code })
+					({ symbol }
+					{ symbol !== code && ' ' + code })
```

Comment:
(minor) not sure if it's more or less readable, but this could ensure that is all in one line:
```suggestion
					({ `${ symbol }${ symbol === code ? '' : ` ${ code }` }` })
```

Reply:
Thanks @frosso! I addressed it in 0f9f22c with a similar solution, just without the main interpolation to avoid too much noise.